packages/common/core-utils/README.md

```diff
@@ -3,10 +3,37 @@
 This package is intended for sharing and promoting best-practice implementations of Fluid-agnostic utility functions
 across packages in the Fluid Framework repo.
 
-<!-- AUTO-GENERATED-CONTENT:START (README_PACKAGE_SCOPE_NOTICE:scopeKind=INTERNAL) -->
+<!-- AUTO-GENERATED-CONTENT:START (LIBRARY_PACKAGE_README_HEADER) -->
 
-**IMPORTANT: This package is intended strictly as an implementation detail of the Fluid Framework and is not intended for public consumption.**
-**We make no stability guarantees regarding its APIs.**
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated using @fluid-tools/markdown-magic. Do not update these generated contents directly. -->
+
+## Using Fluid Framework libraries
+
+When taking a dependency on a Fluid Framework library's public APIs, we recommend using a `^` (caret) version range, such as `^1.3.4`.
+While Fluid Framework libraries may use different ranges with interdependencies between other Fluid Framework libraries,
+library consumers should always prefer `^`.
+
+If using any of Fluid Framework's unstable APIs (for example, its `beta` APIs), we recommend using a more constrained version range, such as `~`.
+
+## Installation
+
+To get started, install the package by running the following command:
+
+```bash
+npm i @fluidframework/core-utils
+```
+
+## Importing from this package
+
+This package leverages [package.json exports](https://nodejs.org/api/packages.html#exports) to separate its APIs by support level.
+For more information on the related support guarantees, see [API Support Levels](https://fluidframework.com/docs/build/releases-and-apitags/#api-support-levels).
+
+To access the `public` ([SemVer](https://semver.org/)) APIs, import via `@fluidframework/core-utils` like normal.
+
+To access the `legacy` APIs, import via `@fluidframework/core-utils/legacy`.
```

Comment:
Nit, not necessarily for this PR: if we can think of some solution to not provide these docs here I think it'd be better, since core-util is explicitly documented as "do not import from this".

Reply:
It is? This package lives in the `@fluidframework` namespace. If that was the case, I would expect it to be `@fluid-internal`.

Reply:
It probably should be, you're right. It has no public exports. @tylerbutler when we created it, maybe we didn't have the `@fluid-internal` story ready? Or was there another reason why this is `@fluidframework`?